File: Data Structures & Algorithms/meeting-schedule/submission-0.py
"""
Definition of Interval:
class Interval(object):
    def __init__(self, start, end):
        self.start = start
        self.end = end
"""

class Solution:
    def canAttendMeetings(self, intervals: List[Interval]) -> bool:
        n = len(intervals)
        intervals.sort(key = lambda x : (x.start, x.end) )

        for i in range(n-1):
            cur, nxt = intervals[i], intervals[i+1]
            if nxt.start < cur.end:
                return False
        
        return True

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        var size = nums.Length;
        var nxt = 1;
        for (int i = 1; i < size; i++)
        {
            if (nums[i] != nums[nxt - 1])
            {
                (nums[i], nums[nxt]) = (nums[nxt], nums[i]);
                nxt += 1;
            }
        }
        return nxt;
    }
}